src/Uno.UI/UI/Xaml/Window.macOS.cs

```diff
@@ -114,6 +137,22 @@ internal void RaiseNativeSizeChanged(CGSize size)
 			}
 		}
 
+		internal void DisplayFullscreen(UIElement element)
```

Comment:
It's actually not really "full screen" but "full window". Probably sufficient for now, but I don't known if it would be easy to really go full screen on mac ?